Remotion/Web/Core/Configuration/ResourceRoot.cs

```diff
@@ -0,0 +1,30 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using Remotion.ServiceLocation;
+
+namespace Remotion.Web.Configuration
+{
+  /// <summary> Default configuration entry specifying the default resource root. </summary>
+  /// <include file='..\doc\include\Configuration\ResourceRoot.xml' path='ResourceRoot/Class/*' />
+  [ImplementationFor(typeof(ResourceRoot), Lifetime = LifetimeKind.Singleton)]
+  public class ResourceRoot
+  {
```

Comment:
please add a public default ctor. It helps to make it more epxlicit that this is the intention and that there's just one ctor.